dist/gmusic-theme.js

```diff
@@ -1008,155 +1013,155 @@ Converter.prototype.getValues = function(space) {
 
 module.exports = convert;
 },{"./conversions":3}],5:[function(require,module,exports){
-module.exports = {
```

Comment:
I can't replicate this diff :cry: 

I even tried on a mac... I can't even visually tell the difference between the two versions.  Is it a line endings thing or what?